.credo.exs

```diff
@@ -173,9 +173,12 @@
         {Credo.Check.Refactor.AppendSingleItem, []},
         {Credo.Check.Refactor.DoubleBooleanNegation, []},
         {Credo.Check.Refactor.ModuleDependencies, false},
-        {Credo.Check.Refactor.NegatedIsNil, []},
+        {Credo.Check.Refactor.NegatedIsNil,
+         files: %{
+           excluded: ["lib/elixir_google_scraper_web/controllers/user_confirmation_controller.ex"]
```

Comment:
Unless we are including/excluding a group of files, I think it might be easier to just disable on that specific line. What do you think?